src/sentry/seer/anomaly_detection/store_data.py

```diff
@@ -49,6 +51,21 @@ def format_historical_data(data: SnubaTSResult) -> list[TimeSeriesPoint]:
     return formatted_data
 
 
+# return the start and end indices
+def _get_start_and_end(data: SnubaTSResult) -> tuple[int, int]:
```

Comment:
If we're returning indices, can we update the name to reflect that?
Also the note can go into a docstring `"""` so it shows up on hover